net/net-mysql-client.cpp

```diff
@@ -271,10 +271,12 @@ static int sqlc_inner_authorise (struct connection *c) {
 
   T = (struct mysql_auth_packet_end *)p;
 
-  const char *sql_username = SQLC_FUNC(c)->sql_get_username(c);
   const char *sql_database = SQLC_FUNC(c)->sql_get_database(c);
+  const char *sql_user = SQLC_FUNC(c)->sql_get_user(c);
   const char *sql_password = SQLC_FUNC(c)->sql_get_password(c);
 
+  kprintf("mysql db: %s; user: %s; password: *\n", sql_database, sql_user);
```

Comment:
```suggestion
  vkprintf(1, "mysql db: %s; user: %s; password: *\n", sql_database, sql_user);
```
It's logging with verbosity set to >= 1